semantic-convention-utils/src/main/java/org/hypertrace/semantic/convention/utils/rpc/RpcSemanticConventionUtils.java

```diff
@@ -24,16 +26,19 @@
   private static final String OTEL_RPC_SYSTEM = OTelRpcSemanticConventions.RPC_SYSTEM.getValue();
   private static final String OTEL_RPC_METHOD = OTelRpcSemanticConventions.RPC_METHOD.getValue();
   private static final String OTEL_GRPC_STATUS_CODE = OTelRpcSemanticConventions.GRPC_STATUS_CODE.getValue();
+  private static final String RPC_STATUS_CODE = OTelRpcSemanticConventions.RPC_STATUS_CODE.getValue();
   private static final String OTEL_RPC_SYSTEM_GRPC = OTelRpcSemanticConventions.RPC_SYSTEM_VALUE_GRPC.getValue();
 
   private static final String OTHER_GRPC_HOST_PORT = RawSpanConstants.getValue(Grpc.GRPC_HOST_PORT);
   private static final String OTHER_GRPC_METHOD = RawSpanConstants.getValue(Grpc.GRPC_METHOD);
-  private static final String[] OTHER_GRPC_STATUS_CODES =
-      {
+  private static final List<String> ALL_GRPC_STATUS_CODES =
+      List.of(
           RawSpanConstants.getValue(CensusResponse.CENSUS_RESPONSE_STATUS_CODE),
           RawSpanConstants.getValue(Grpc.GRPC_STATUS_CODE),
-          RawSpanConstants.getValue(CensusResponse.CENSUS_RESPONSE_CENSUS_STATUS_CODE)
-      };
+          RawSpanConstants.getValue(CensusResponse.CENSUS_RESPONSE_CENSUS_STATUS_CODE),
+          RPC_STATUS_CODE,
```

Comment:
Just a note: The existing order was `RPC_STATUS_CODE, RawSpanConstants.getValue(CensusResponse.CENSUS_RESPONSE_CENSUS_STATUS_CODE)`. But, I am assuming that should not be a problem.

Reply:
Will do